.storybook/stories/datagrid/datagrid-row.stories.ts

```diff
@@ -40,7 +40,7 @@ const defaultStory: Story = args => ({
       <clr-dg-row
         *clrDgItems="let element of elements; let index = index"
         [clrDgExpanded]="clrDgExpanded && index === 0"
-        [clrDgSelectable]="clrDgSelectable && index === 0"
```

Comment:
We should consider adding docblocks in storybook to give better descriptions to what the user is expected to see with our parameters. It's not apparent that most of these are designed to only apply to the first row.